src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/ScriptActionsOnRunningCluster.cs

```diff
@@ -122,7 +122,7 @@ public void TestScriptActionsOnRunningCluster()
                     var result = client.Clusters.ExecuteScriptActions(resourceGroup, dnsName, executeScriptActionParams);
                     Assert.Equal(result.StatusCode, HttpStatusCode.OK);
                     Assert.Equal(result.State, AsyncOperationState.Failed);
-                    Assert.Equal(result.ErrorInfo.Message, "ScriptExecutionFailed");
```

Comment:
The latest from the dev who owns fixing this issue

Hi Rajesh,

The fix for this bug is already checked in (changeset 990754) on 09/27. We don’t have a work item tracking this bug. I was not able to reproduce the same issue in current or dogfood, and it did not repro in prod either the very next day when RP hotfix was pushed to PROD. 

I have created a work item to add the validation back to SDK. http://sqlbuvsts01:8080/Main/HDInsight/_workitems#id=8668414&_a=edit Will get to it asap.

Thanks,
Namrata